tools/cxbe/Main.cpp

```diff
@@ -19,19 +20,33 @@ int main(int argc, char *argv[])
     char szXbeFilename[OPTION_LEN + 1] = { 0 };
     char szDumpFilename[OPTION_LEN + 1] = { 0 };
     char szXbeTitle[OPTION_LEN + 1] = "Untitled";
+    char szXbeTitleID[OPTION_LEN + 1] = "";
+    char szXbeRegions[OPTION_LEN + 1] = "";
+    char szXbeVersion[OPTION_LEN + 1] = "";
     char szMode[OPTION_LEN + 1] = "retail";
     char szLogo[OPTION_LEN + 1] = "";
+
     bool bRetail;
+    uint32 dwTitleId = 0x4358270F; // CX-9999
```

Comment:
nit: I'd suggest to stick to 0xFFFF0002 which is more obviously a fake ID and has been used for years in Cxbe. If we add CX-9999 it will just taint another valid XBE Title-ID.